src/patternfly/components/Page/page.scss

```diff
@@ -258,35 +257,20 @@ $pf-c-page--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "xl",
 .pf-c-page__header-tools-item {
   @include pf-hidden-visible(var(--pf-c-page__header-tools-item--Display));
 
-  .pf-c-button {
```

Comment:
This causes selected, non-notification badge buttons to appear behind the selected grey background.